src/plugins/custom_integrations/common/index.ts

```diff
@@ -56,6 +56,8 @@ export interface CustomIntegration {
   icons: Array<{ src: string; type: string }>;
   categories: Category[];
   shipper: string;
+  eprOverlap?: string;
 }
 
-export const ROUTES_ADDABLECUSTOMINTEGRATIONS = `/api/${PLUGIN_ID}/appendCustomIntegrations`;
+export const ROUTES_APPEND_CUSTOM_INTEGRATIONS = `/api/${PLUGIN_ID}/appendCustomIntegrations`;
+export const ROUTES_REPLACEMENT_CUSTOM_INTEGRATIONS = `/api/${PLUGIN_ID}/replacementCustomIntegrations`;
```

Comment:
Since these are internal APIs and are subject to change, we should use the `/internal` prefix to follow Kibana's API convention.